Yoast/wordpress-seo
# Add reindex link notice

## Summary

This PR can be summarized in the following changelog entry:
Added notification to let the user know there is something to reindex.

## Relevant technical choices:

* Added notifier object to add notice to the dashboard

## Test instructions

This PR can be tested by following these steps:

* Have something to reindex
* See a notice will pop up. You might do something with the scheduled cron.

See #7294
